### PR TITLE
Misc MI docs update

### PR DESCRIPTION
**Pull request by michaellzc**

### CODE REVIEW

**Comment by michaellzc** [comment]
Current dependencies on/for this PR:
* main
  * **PR #4715** <a href="https://app.graphite.dev/github/pr/sourcegraph/handbook/4715" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/sourcegraph/handbook/4715?utm_source=stack-comment).

File: content/departments/cloud/technical-docs/operations.md [hunk 1]
```diff
@@ -483,6 +465,43 @@ service:
...
+The global override files are separated into different files by purpose. When adding a new override, you should consider whether you should create a new one or append your override in an existing file. Upon updating the global override files, you should run the command below to update the golden files. The `update-golden` command will pull the target file from upstream, then merge it with every global override files.
+
+> NOTES: you should never edit the golden files manually, let `mg` CLI to do the work
+
+```sh
+mg update-golden -target $version
+```
+
+Commit your change and make a PR. Once the PR is merged, you can follow [this process](#deploy-new-images-across-all-instances) to roll out changes to all instances.
+
```
Comment (michaellzc) [inline content/departments/cloud/technical-docs/operations.md:504]: @filiphaftek the docs about how to update golden files

File: content/departments/cloud/technical-docs/operations.md [hunk 2]
```diff
@@ -483,6 +465,43 @@ service:
...
+- Click `Run workflow` and it will run `mg sync` on each instance
+
+> This action also runs every 24h to ensure all instances config are correct
+
+### Make global changes to docker-compose.yaml across all deployment
+
+Use case;
+
+- Configure default `cpus` of `gitserver` for all instances
+- Use a custom `prometheus` image for all instances
```
Comment (danieldides) [inline content/departments/cloud/technical-docs/operations.md:491]: 👀

**Review by danieldides — APPROVED** [review]
This is so awesome.

**Comment by filiphaftek** [comment]
Thanks a lot @michaellzc ! This is super helpful :)

**Pull request merged**